src/org/opendatakit/briefcase/util/WebUtils.java

```diff
@@ -103,6 +107,46 @@
         connectionManager.setDefaultMaxPerRoute(MAX_CONNECTIONS_PER_ROUTE);
     }
 
+    /**
+     * Get the {@link HttpClientContext} bound to the current thread.
```

Comment:
I like 3rd person declarative rather than 2nd person imperative, e.g., “gets”, not “get”.